src/NuGet.Indexing/Extraction/CatalogPackageReader.cs

```diff
@@ -87,5 +87,20 @@ public override Task<byte[]> GetArchiveHashAsync(HashAlgorithmName hashAlgorithm
         {
             throw new NotImplementedException();
         }
+
+        public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verifierSettings)
+        {
+            throw new NotImplementedException();
```

Comment:
Shouldn't this be `false` instead?

Reply:
It is never called neither in our code, nor in the client library, so shouldn't matter. I will change though.